lib/daru/index/multi_index.rb

```diff
@@ -171,8 +185,20 @@ def find_all_indexes label, level_index, chosen
       end
     end
 
+    # Array `name` must have same length as levels and labels.
+    def validate_name names, levels
+      error_msg = 'names and levels should be of same size. size of the name '\
+      'array is #{names.size} and size of the MultiIndex levels and labels is'\
+      ' #{labels.size}.'
```

Comment:
Thanks @zverok for pointing it out. Actually at first I was using double quote but rubocop returned error `Style/StringLiterals`, then I used single quote (which was wrong). I have corrected it in this commit https://github.com/SciRuby/daru/pull/323/commits/5955197912cd83223c9f790c9e4264389a0348e4